gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Mar 11 13:43:48 CET 2024
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Why not using the latest version of gradle? (8.10.2)

If upgrading is compatible with the project, please run `.\gradlew wrapper --gradle-version 8.10.2 `